### PR TITLE
Remove jQuery utils

### PR DESCRIPTION
**Pull request by bautrukevich**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -179,61 +201,69 @@ const applyCropCoordsToInfo = function(info, crop, size, coords) {
...
+    input.type = 'file'
+
+    if (settings.multiple) {
+      input.setAttribute('multiple', '')
+    }
+
+    input.setAttribute('accept', accept)
+
+    input.addEventListener('change', function() {
+      fn(this)
```
Comment (jeetiss) [inline src/utils.js]: what is `this`?

File: src/utils.js [hunk 2]
```diff
@@ -302,39 +332,21 @@ const ajaxDefaults = {
   cache: false
 }
 
-const jsonp = function(url, type, data, settings = {}) {
```
Comment (jeetiss) [inline src/utils.js]: why did you remove `jsonp`? code still use it in files directory

File: src/utils.js [hunk 3]
```diff
@@ -46,23 +44,47 @@ const once = function(fn) {
...
 const wrapToPromise = function(value) {
-  return $.Deferred()
-    .resolve(value)
-    .promise()
+  return Promise.resolve(value)
+}
+
+// same as promise.then(), but if filter returns promise
+// it will be just passed forward without any special behavior
+const then = function(pr, doneFilter, failFilter, progressFilter) {
```
Comment (jeetiss) [inline src/utils.js]: Why did you return `then`? I removed it already

File: src/utils.js (hunk 1 above)
Comment (bautrukevich) [reply src/utils.js]: HTMLInputElement, right?

File: src/utils.js (hunk 3 above)
Comment (bautrukevich) [reply src/utils.js]: Yes, you are right, sorry

File: src/utils.js (hunk 2 above)
Comment (bautrukevich) [reply src/utils.js]: Because we don't need this, it's all about small PR's, I remove $ from utils

File: src/utils.js (hunk 1 above)
Comment (bautrukevich) [reply src/utils.js]: Fixed

File: src/utils.js (hunk 2 above)
Comment (jeetiss) [reply src/utils.js]: move it like in #690

Comment (bautrukevich) [reply src/utils.js]: Ok, done
